lib/orogen/templates/main.cpp

```diff
@@ -168,6 +169,12 @@ int ORO_main(int argc, char* argv[])
    }
 
 
+   const char *ldLibPath = getenv("LD_LIBRARY_PATH");
```

Comment:
as i see this is not completly true, the LD path is only valid for local install. But the libs could be also installted in /lib or /include/lib (global lib paths). Therefore setting this fixed to the extended one feels wrong for general installations